### DISCUSSION TITLE
IBM Z Open Editor V4 and dependency on other products for licensed features

### DISCUSSION THREAD

**Discussion by FALLAI-Denis** [IBM/zopeneditor-about#395] [General]
Hi,

IBM Z Open Editor V4 now offers features under license control.

I myself suggested this direction because it seems to me that it is a certain form of guarantee of the sustainability of the solution and the possibility of having functionalities which exist today in the paid solution IDz.

On the other hand, from what I have read in the [presentation of these new features](https://ibm.github.io/zopeneditor-about/Docs/advanced_overview.html), access to licensed features requires the acquisition of other products, and this is a serious problem and constraint that we will not be able to satisfy:
* [IBM Developer for z/OS Enterprise Edition](https://www.ibm.com/products/developer-for-zos)
* [IBM Application Delivery Foundation for z/OS](https://www.ibm.com/products/app-delivery-foundation-for-zos).

For our part, we do not intend to acquire new products, or if we do it will not be in relation to enabling features in IBM Z Open Editor, and if we have chosen to use IBM Z Open Editor in VS Code, it is because we do not want to use (and therefore acquire) IDz.

Furthermore, we have acquired IBM Dependency Based Build, standalone version, and one of the functionalities is linked to IBM DBB, however this does not enable the DBB build log analysis functionality in ZOE, (analysis of compilation listings).

There should therefore be a way to license the functionalities of IBM Z Open Editor independent of any acquisition of another product.

Thanks

**Comment by phaumer** [comment]
Thanks for the feedback @FALLAI-Denis. I will bring your perspective to product management. IDzEE and ADFz are the offerings for purchasing Z Open Editor for a while now, so it was the straight-forward way for us to enable the new features through these. These offerings are not limited to Eclipse-based technologies anymore and will be a key vehicle for providing next generation and VS Code based developer experiences to our customers.
